File: app/src/main/java/es/etologic/mahjongscoring2/domain/entities/BestHand.java
package es.etologic.mahjongscoring2.domain.entities;

public class BestHand {

    private String playerName;
    private int handValue;

    public String getPlayerName() {
        return playerName;
    }

    public int getHandValue() {
        return handValue;
    }

    public void setPlayerName(String playerName) {
        this.playerName = playerName;
    }

    public void setHandValue(int handValue) {
        this.handValue = handValue;
    }

    BestHand() {
        playerName = "";
        handValue = 0;
    }
}

File: app/src/main/java/com/etologic/mahjongscoring2/business/model/dtos/BestHand.kt
package com.etologic.mahjongscoring2.business.model.dtos

import com.etologic.mahjongscoring2.business.model.enums.TableWinds
import com.etologic.mahjongscoring2.business.model.enums.TableWinds.NONE

class BestHand {
    
    var playerInitialPosition: TableWinds = NONE
    var playerName: String = ""
    var handValue: Int = 0
    
}
